src/main/java/com/arpnetworking/metrics/impl/BaseFileSink.java

```diff
@@ -52,21 +54,20 @@ protected Logger getMetricsLogger() {
         return _metricsLogger;
     }
 
-    private SizeAndTimeBasedRollingPolicy<ILoggingEvent> createRollingPolicy(
+    private TimeBasedRollingPolicy<ILoggingEvent> createRollingPolicy(
             final String extension,
             final String fileNameWithoutExtension,
             final int maxHistory,
             final String maxFileSize,
             final String totalSizeCap,
             final boolean compress) {
 
-        // TODO(vkoskela): Use LogbackSteno's random compression strategy [MAI-413]
```

Comment:
You didn't actually do this. There are still benefits to using the random offset rotation to avoid concurrent rotation of several files on the hour. If you don't want to do this now, that's fine, but I think we should still consider this for the future (e.g. leave the TODO); however, feel free to remove the Groupon issue reference.